src/app/components/SubjectHeading/SubjectHeading.jsx

```diff
@@ -211,6 +211,10 @@ class SubjectHeading extends React.Component {
             <Preview topHeadings={preview}/>
             : null
           }
+          {container === 'narrower' ?
+            <hr className="relatedHeadingsBoundary"/>
+            : null
+          }
```

Comment:
Great idea and implementation!
Just tried moving lines 214-217 to the beginning of the li and it fixes the placement so that there is one between the column headings and the subject headings and not one at the very bottom. We can use CSS first child selector to style that first one distinctly.